internal/dockerfile/dockerfile_test.go

```diff
@@ -174,9 +174,7 @@ func TestFindImagesWeirdSyntax(t *testing.T) {
 	df := Dockerfile(`FROM a b`)
 	images, err := df.FindImages()
 	assert.NoError(t, err)
-	if assert.Equal(t, 1, len(images)) {
-		assert.Equal(t, "docker.io/library/a", images[0].String())
-	}
+	assert.Equal(t, 0, len(images))
```

Comment:
hmm, why did this change?

Reply:
good question. i went back and forth on this. it's a question of how robustly we handle parse errors. i changed my mind and changed it back!